ujenkins/endpoints/builds.py

```diff
@@ -35,20 +64,36 @@ def get(self, name: str) -> List[dict]:
             ]
 
         Args:
-            name (str):
-                Job name or path (if in folder).
+            name (str): Job name or path (if in folder).
+            fields (Optional[Union[List[str], str]]): List of fields to return.
+                Possible values are mentioned in the available fields.
+                If empty, default fields will be returned.
+            start (int, optional): The start index of the builds to retrieve.
+                Used with the 'end' parameter to specify a range.
+                Defaults to None.
+            end (int, optional): The end index of the builds to retrieve.
+                Used with the 'start' parameter to specify a range.
+                Defaults to None.
 
         Returns:
-            List: list of build for specified job.
+            List[dict]: list of builds for specified job.
         """
+
+        page_string = ''
+        if end is not None or start is not None:
+            start_str = str(start) if start is not None else ''
```

Comment:
you don't need to cast it from int to str, because it will be interpolated in f-string nicely anyway